STM32Project/Core/Inc/projectConstants.h

```diff
@@ -103,7 +174,8 @@ extern int ACCEL_CCR_FOR_DUTY_CYCLE_MAX;
 #define SQUARE CONTROLLER_VALUES[5]
 #define TRIANGLE CONTROLLER_VALUES[6]
 #define L1 CONTROLLER_VALUES[7]
-extern int CONTROLLER_VALUES[8];
+#define R_JOYSTICK CONTROLLER_VALUES[8]
+extern int CONTROLLER_VALUES[9];
```

Comment:
Why did we move Controller Values to index 9? Are we accepting an extra input we didn't before?